src/resources/armies/moloch.json

```diff
@@ -0,0 +1,332 @@
+{
+  "id": "moloch",
+  "name": "Moloch",
+  "tokens": [
+    {
+      "id": "hq",
+      "hq": true,
+      "initiative": 0,
+      "attacks": [
+        {
+          "type": "riffle",
+          "angle": -1
+        }
+      ],
+      "modifiers": [
+        {
+          "type": "strengthr",
+          "angle": -1,
+          "value": 1,
+          "self": true
```

Comment:
I guess we want to invert this by replacing "self" with "hostile", as most of the modifiers actually apply to allies only.